resource-managers/kubernetes/core/src/main/scala/org/apache/spark/deploy/kubernetes/constants.scala

```diff
@@ -92,25 +92,20 @@ package object constants {
 
   // V2 submission init container
   private[spark] val INIT_CONTAINER_ANNOTATION = "pod.beta.kubernetes.io/init-containers"
-  private[spark] val INIT_CONTAINER_SECRETS_VOLUME_NAME = "dependency-secret"
-  private[spark] val INIT_CONTAINER_SECRETS_VOLUME_MOUNT_PATH = "/mnt/secrets/spark-init"
-  private[spark] val INIT_CONTAINER_DOWNLOAD_JARS_SECRET_KEY = "downloadJarsSecret"
-  private[spark] val INIT_CONTAINER_DOWNLOAD_FILES_SECRET_KEY = "downloadFilesSecret"
-  private[spark] val INIT_CONTAINER_TRUSTSTORE_SECRET_KEY = "trustStore"
-  private[spark] val INIT_CONTAINER_DOWNLOAD_JARS_SECRET_PATH =
-    s"$INIT_CONTAINER_SECRETS_VOLUME_MOUNT_PATH/$INIT_CONTAINER_DOWNLOAD_JARS_SECRET_KEY"
-  private[spark] val INIT_CONTAINER_DOWNLOAD_FILES_SECRET_PATH =
-    s"$INIT_CONTAINER_SECRETS_VOLUME_MOUNT_PATH/$INIT_CONTAINER_DOWNLOAD_FILES_SECRET_KEY"
-  private[spark] val INIT_CONTAINER_TRUSTSTORE_PATH =
-    s"$INIT_CONTAINER_SECRETS_VOLUME_MOUNT_PATH/$INIT_CONTAINER_TRUSTSTORE_SECRET_KEY"
-  private[spark] val INIT_CONTAINER_DOWNLOAD_CREDENTIALS_PATH =
-    "/mnt/secrets/kubernetes-credentials"
-  private[spark] val INIT_CONTAINER_CONFIG_MAP_KEY = "init-driver"
-  private[spark] val INIT_CONTAINER_PROPERTIES_FILE_VOLUME = "init-container-properties"
-  private[spark] val INIT_CONTAINER_PROPERTIES_FILE_MOUNT_PATH = "/etc/spark-init/"
-  private[spark] val INIT_CONTAINER_PROPERTIES_FILE_NAME = "init-driver.properties"
+  private[spark] val INIT_CONTAINER_SECRET_VOLUME_MOUNT_PATH =
+    "/mnt/secrets/spark-init"
+  private[spark] val INIT_CONTAINER_SUBMITTED_JARS_SECRET_KEY =
+    "downloadSubmittedJarsSecret"
+  private[spark] val INIT_CONTAINER_SUBMITTED_FILES_SECRET_KEY =
```

Comment:
They're only separate secret keys. The reason is simply for architectural ease. We want to deploy the files bundle and the jars bundle in different directories, so we just upload two bundles for each resource type and download them to different directories.